tasks/destroy.py

```diff
@@ -17,7 +17,7 @@ def destroy(
     scenario_name: str,
     config_path: Optional[str] = None,
     stack: Optional[str] = None,
-    force_yes: Optional[bool] = False,
+    force_yes: Optional[bool] = True,
```

Comment:
What about we just remove that params then? I would be surprised that someone ever use the `--no-force-yes` flag

Reply:
Done!